<launch> 

<param name="state" value="0" /> 
<param name="TimeforSleeping" value="10" /> 
<param name="TimetoGetPosition" value="4" /> 
<param name="WaitForANewPointingGesture" value="5" /> 


<node name="state_machine" pkg="assignment1_ExRoLab" type="state_machine.py" output="screen"/> 
<node name="display" pkg="assignment1_ExRoLab" type="Display.py" output="screen"/> 

</launch> 
